src/function/pragma/pragma_queries.cpp

```diff
@@ -19,19 +19,32 @@ string PragmaTableInfo(ClientContext &context, const FunctionParameters &paramet
 	                          KeywordHelper::WriteQuoted(parameters.values[0].ToString(), '\''));
 }
 
-string PragmaShowTables() {
+string PragmaShowTables(const string &database, const string &schema) {
+	string where_clause = "";
+	vector<string> where_conditions;
+	if (!database.empty()) {
+		where_conditions.push_back("database_name = '" + database + "'");
```

Comment:
Can we use `StringUtil::Format` with `SQLString(database)` as parameters so that these are properly escaped?

Reply:
Can we also add a test with a database name that has quotes in it?